developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -177,19 +198,18 @@ import ManageDataCodeV3 from '!!raw-loader!/_includes/code/howto/manage-data.rea
   </TabItem>
 </Tabs>
 
-### Collections, classes and schemas
+### Nomenclature changes
 
-The focus on [collections](/developers/weaviate/config-refs/schema) in the v4 client is related to some corresponding changes in Weaviate Core:
+Some of the nomenclature within the Weaviate ecosystem is changing, and the client nomenclature has changed accordingly:
```

Comment:
Consider a more common word:

Weaviate is updating it's terminology. The client reflects the changes: